### ISSUE TITLE
In macOS 10.13.6 plus security update 2018-003 patch is not working...

### ISSUE THREAD

**Issue by base08** [aonez/NightShiftPatcher#9]
Hi @aonez ,

Thanks for your work!

I've recently updated to the latest security update and I get the:
`The new signature is invalid or can't be applied. Check the original framework sigature is valid and try again. No patch applied.`

This is a copy of the whole output:
`NightShiftPatcher 1.0.0 by aONe ® 2017-2018 (https://github.com/aonez/NightShiftPatcher)
Original idea by Pike (https://pikeralpha.wordpress.com/2017/01/30/4398)
Motivated by NightPatch (https://github.com/pookjw/NightPatch)

Checking for requirements...
SIP is disabled
Binary nm is functional
Looking for minimum requirements check offset...
Offset: 0x44670
Getting offset hex data...
Original hex: 00044670: 0900 0000 0100 0000 0d00 0000 0600 0000 0500 0000 0600 0000  ........................
Cloning the framework...
/System/Library/PrivateFrameworks/CoreBrightness.framework.hack
/System/Library/PrivateFrameworks/CoreBrightness.framework.hack/Versions/A/CoreBrightness
Replacing offset hex data...
24+0 records in
24+0 records out
24 bytes transferred in 0.000334 secs (71902 bytes/sec)
Checking offset hex replaced...
Replaced hex: 00044670: 0100 0000 0100 0000 0100 0000 0100 0000 0100 0000 0100 0000  ........................
Resigning the framework...
/System/Library/PrivateFrameworks/CoreBrightness.framework.hack: replacing existing signature
/System/Library/PrivateFrameworks/CoreBrightness.framework.hack: code object is not signed at all
In subcomponent: /System/Library/PrivateFrameworks/CoreBrightness.framework.hack/Versions/Current/CoreBrightness.tbd
Checking new signature...

The new signature is invalid or can't be applied. Check the original framework sigature is valid and try again. No patch applied.
/System/Library/PrivateFrameworks/CoreBrightness.framework.hack: invalid signature (code or signature have been modified)
In architecture: x86_64`

Thanks!

**Comment by aonez** [comment]
Hi @base08!

Can you check if the script you're using has this lines:

https://github.com/aonez/NightShiftPatcher/blob/1b3db5516cb756b96abb7dbabc11fb82640bd04b/NightShiftPatcher.sh#L118-L121

If it does not, please download the script again because is an old version. I've just updated the versioning of the script to 1.0.1 because I've fixed some things over the time but never updated the version 😅

If the script already has that lines or the newly downloaded one fails too, execute this and try again:
```sh
sudo rm /System/Library/PrivateFrameworks/CoreBrightness.framework/Versions/Current/CoreBrightness.tbd
```

**Comment by base08** [comment]
Ah! Worked!

I've manually pasted those lines into your code and voila everything is perfect again!

Thanks for the quick response!

**Comment by aonez** [comment]
Just released the version [1.0.1](https://github.com/aonez/NightShiftPatcher/releases/tag/v1.0.1) to make clear there're changes in the script 😊

Thanks for the feedback @base08!
